AirCasting/SessionViews/StreamsView.swift

```diff
@@ -0,0 +1,67 @@
+// Created by Lunar on 29/06/2021.
+//
+
+import SwiftUI
+
+struct StreamsView: View {
+    
+    @Binding var selectedStream: MeasurementStreamEntity?
+    @ObservedObject var session: SessionEntity
+    var threshold: SensorThreshold
+    @EnvironmentObject private var microphoneManager: MicrophoneManager
+    let showValues: Bool
```

Comment:
also Can this be private too? 😇

Reply:
Unfortunately nope, because it's then inaccessible in AirMapViewm GraphView and SessionCartView :)